aws/import_aws_api_gateway_stage_test.go

```diff
@@ -0,0 +1,27 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSApiGatewayStage_importBasic(t *testing.T) {
+	resourceName := "tf-acc-test.dev"
```

Comment:
This should be a reference to a resource in a test configuration, for `testAccAWSAPIGatewayStageConfig_basic` its resource definition is:

```hcl
resource "aws_api_gateway_stage" "test" {
  rest_api_id = "${aws_api_gateway_rest_api.test.id}"
  stage_name = "prod"
  deployment_id = "${aws_api_gateway_deployment.dev.id}"
  cache_cluster_enabled = true
  cache_cluster_size = "0.5"
  variables {
    one = "1"
    two = "2"
  }
  tags {
    Name = "tf-test"
  }
}
```

So this should be `resourceName := "aws_api_gateway_stage.test"`.

It also looks that we require name randomization for `testAccAWSAPIGatewayStageConfig_basic()` now. You can create a variable `rName := acctest.RandomWithPrefix("tf-acc-test")` and call that function as `testAccAWSAPIGatewayStageConfig_basic(rName)`